shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1652,15 +1653,17 @@ public void writeFields(DataOutput out) throws IOException {
             out.writeUTF(segment);
             out.writeUTF(delegationToken == null ? "" : delegationToken);
             tableEntries.writeFields(out);
+            out.writeLong(tableSegmentOffset);
         }
 
         public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOException {
             long requestId = in.readLong();
             String segment = in.readUTF();
             String delegationToken = in.readUTF();
             TableEntries entries = (TableEntries) TableEntries.readFrom(in, in.available());
+            long tableSegmentOffset = in.readLong();
```

Comment:
we should read this value only if it available.Else the OLDEST_COMPATIBLE_VERSION will change.